app/src/actions/modal.js

```diff
@@ -0,0 +1,39 @@
+import {
```

Comment:
let's rename this file to welcomeModal.js
This project has a lot of modal, and us-in-6-months will get confused by the simplistic name here.